telegram/webhookinfo.py

```diff
@@ -40,13 +40,20 @@ class WebhookInfo(TelegramObject):
 
     """
 
-    def __init__(self, url, has_custom_certificate, pending_update_count, **kwargs):
+    def __init__(self,
+                 url,
+                 has_custom_certificate,
+                 pending_update_count,
+                 last_error_date=None,
+                 last_error_message=None,
```

Comment:
I thought that too, so I checked the official December 4th release notes and it wasn't there, so lets fix it here while we're at it.